packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -95,4 +95,30 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
 
     return mapSchemasValues(schemas);
   }
+
+  async getConfigurationStream(): Promise<Writable> {
+    if (!this.strapi) {
+      throw new Error('Not able to stream Configurations. Strapi instance not found');
+    }
+
+    const strapi = this.strapi;
+
+    return new Writable({
+      objectMode: true,
+      async write(config, _encoding, callback) {
+        try {
+          await restoreConfigs(strapi, config);
```

Comment:
Shouldn't check we're using a restore strategy before calling the `restoreConfigs`?

Reply:
hmmmm, yes indeed